app/src/main/java/org/simple/clinic/login/OtpSmsReceiver.kt

```diff
@@ -51,6 +53,8 @@ class OtpSmsReceiver : BroadcastReceiver() {
         // TODO: Schedule call with worker instead of logging in directly
         userSession.loginWithOtp(otp)
             .subscribeOn(io())
+            // TODO: Remove later after feature is tested
```

Comment:
We need this because it is very difficult to test manual OTP entry on staging if auto read is enabled.

Reply:
Will do this next time I need to do something like this. 👍